docs/learn/learn-polkadot-opengov.md

```diff
@@ -530,6 +530,28 @@ Note that to successfully cast votes you need to have the
 [existential deposit](./learn-accounts.md#existential-deposit-and-reaping) and some additional funds
 to pay for transaction fees.
 
+### Voting Locks
+
+A lock is placed on the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} when you vote on an
+Ongoing referendum. If you choose to remove your vote when the referendum is still in the decision period, 
+your voting lock is removed. Once the referendum is decided (accepted or rejected), your {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
+will be available for unlocking if you voted with zero conviction. The governance app or interface you 
+used for participating in Polkadot OpenGov should show an option to unlock your {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}.
+[Here](https://support.polkadot.network/support/solutions/articles/65000184129) are the instructions to 
+do it on the Polkadot-JS UI.
+
```

Comment:
```suggestion

#### Voting with Conviction

```

Reply:
@DrW3RK I added two subtitles. I will refine this section and the delegation section in another PR.

Reply:
Sounds good. Thank you!